test/unit/identity/interaction/email-password/handler/Util.ts

```diff
@@ -0,0 +1,14 @@
+import { stringify } from 'querystring';
+import type { HttpRequest } from '../../../../../../src/server/HttpRequest';
+import { guardedStreamFrom } from '../../../../../../src/util/StreamUtil';
```

Comment:
```suggestion
import { guardedStreamFrom } from '../../../../../../src/util/StreamUtil';
import { APPLICATION_X_WWW_FORM_URLENCODED } from '../../../../../../src/util/ContentTypes';
```
?

Reply:
Most unit tests actually do not use the constants defined in `ContentTypes`. Whether this is intentional or just because we didn't pay attention in the beginning I'll leave in the middle 😉, but I sort of like that this way we're checking in multiple ways if we definitely use the correct type.

Reply:
This is intentional indeed.

Reply:
Aha, that makes sense :)